core/PRP/Animation/plLeafController.cpp

```diff
@@ -441,7 +441,7 @@ void plLeafController::setKeys(const std::vector<hsKeyFrame*>& keys, unsigned in
     DeallocKeys();
     AllocKeys(keys.size(), type);
     for (size_t i=0; i<keys.size(); i++)
-        fKeys[i] = keys[i];
+        *fKeys[i] = *keys[i];
```

Comment:
Ah, good points.  But we still need `setEaseControllers` to get fixed as well :)